exec.go

```diff
@@ -703,12 +705,31 @@ func tab(et *Text, _ *Text, argt *Text, _, _ bool, arg string) {
 		args := strings.Split(arg, " ")
 		arg = args[0]
 		p := arg
+		if len(p) == 0 {
```

Comment:
I think that there is a much better general fix here. Can you file an issue and add a `TODO(rjk):` to use a tokenizer on the results of `getarg`. Seems like a nice opportunity to refactor.

Reply:
Updated my PR with the comment and created the issue.

Reply:
thanks